tests/models/test_cookies.py

```diff
@@ -30,6 +30,16 @@ def test_cookies_update():
     assert cookies.get("name", domain="example.com") == "value"
 
 
+def test_cookie_value_can_be_none():
+    cookies = httpx.Cookies()
+    more_cookies = httpx.Cookies()
+    more_cookies.set("no-value", None, domain="example.com")
+
+    cookies.update(more_cookies)
+    assert dict(cookies) == {"no-value": None}
+    assert cookies.get("no-value", domain="example.com") is None
```

Comment:
It's not absolutely clear to me that allowing `None` here is an API improvement. 🤔

Reply:
I changed the test to `assert cookies["no-value"] is None`.
I also added `test_get_missing_cookie_raises_exception` to highlight the difference between getting a missing cookie (raises exception) and getting a cookie whose value is `None`.

Is it better?